src/applications/discharge-wizard/tests/v2/DischargeMonth.unit.spec.js

```diff
@@ -0,0 +1,97 @@
+// import React from 'react';
+// import { Provider } from 'react-redux';
+// import { render } from '@testing-library/react';
+// import { expect } from 'chai';
+// import sinon from 'sinon';
+// import { RESPONSES } from '../../constants/question-data-map';
+// import { ROUTES } from '../../constants';
+
+// import DischargeMonth from '../../components/v2/questions/DischargeMonth';
+
+// const mockStoreStandard = {
+//   getState: () => ({
+//     dischargeUpgradeWizard: {
+//       duwform: {
```

Comment:
This should be updated wherever you use it - it's `duwForm`. These syntax errors cause that not helpful generic test failure :(